### ISSUE TITLE
Zmiana opisu, ustawienie statystyki encji

### ISSUE THREAD

**Issue by kzajac83** [tcich/ha-addon-energa-meter#14]
Dzień dobry.

Proponuję do domyślnej konfiguracji readme.md o [tutaj ](https://github.com/tcich/ha-addon-energa-meter/blob/main/src/README.md#konfiguracja-sensor%C3%B3w) dodać do każdej encji dwie wartości:

![image](https://github.com/user-attachments/assets/36fd0c1b-6060-4641-b1eb-48c89659bda7)

Spowoduje to uwzględnienie encji jako danych statystycznych, które można uwzględniać na dashboardzie enegetycznym HA. Więcej info tutaj:

https://gist.github.com/langestefan/acbf7e42b15b302e9aacc08663039e57#2-my-entity-is-not-listed-when-i-try-to-add-it-to-the-energy-dashboard-how-can-i-make-it-visible

**Comment by paki111** [comment]
masz racje, ja sobie dodałem
device_class: energy
state_class: total_increasing

**Comment by kzajac83** [comment]
Tak, zgodnie z wytycznymi `state_class `dla tych dwóch encji powinno być `total_increasing `, a nie  `total` (choć total to też zadziała w tym przypadku tak samo).

Tutaj wytłumaczone dlaczego:

https://developers.home-assistant.io/docs/core/entity/sensor/#available-state-classes

https://developers.home-assistant.io/blog/2021/09/20/state_class_total/
